docs/src/main/sphinx/security/file-system-access-control.md

```diff
@@ -816,7 +805,11 @@ These rules apply to `filter_environment` and `mask_environment`.
 
 #### Function rules
 
-Each function rule is composed of the following fields:
+These rules control the ability of a user to create, drop, and execute functions.
```

Comment:
same idea as above .. catalog routines

Reply:
but actually these are for 

table functions, functions from custom plugins and catalog routines

Reply:
I think it's fine to say "functions" here. The execute is for all functions and the SQL is `CREATE FUNCTION`.

Reply:
Maybe we should add "function" to the glossary or concepts page..

Reply:
Probably concepts when I look at that page - https://trino.io/docs/current/overview/concepts.html